app/controllers/admin/organisations_controller.rb

```diff
@@ -84,6 +84,8 @@ def update
     end
   end
 
+  def confirm_destroy; end
+
   def destroy
     @organisation.destroy!
     redirect_to admin_organisations_path
```

Comment:
Maybe a notice could be added here to inform the user the organisation has been deleted.

Reply:
good shout have added it 👍